app/controllers/games_controller.rb

```diff
@@ -1,4 +1,34 @@
 class GamesController < ApplicationController
   # Add your GamesController code here
+  
+  def create
+      @game = Game.create(state: params[:state])
+      
+    #   render json: @game, status: 201
+    #   render "/home/index.html"
+      render json: @game
+  end 
+  
+  
+  def index 
+      @games = Game.all
+    #   binding.pry
+    #   render "/home/index.html"
+      render json: @games
+  end 
+  
+  def show 
+      @game = Game.find(params[:id])
+    #   render "/home/index.html"
+      render json: @game
+  end 
+  
+  def update 
+      @game = Game.find(params[:id])
+      @game.state = params[:state]
+      @game.save
+    #   render "/home/index.html"
+      render json: @game
+  end 
 
```

Comment:
You should always ensure that you are whitelisting your parameters to only allow approved params to be used in the controller.

```ruby
class GamesController < ApplicationController
  #your game routes

  private
    def game_params
      params.permit(state: [])
    end
end
```